pkg/operators/formatters/formatters.go

```diff
@@ -159,6 +163,39 @@ var replacers = []replacer{
 		},
 		priority: 0,
 	},
+	{
+		name:      "syscall",
+		selectors: []string{"type:" + SyscallTypeName},
+		replace: func(ds datasource.DataSource, in datasource.FieldAccessor) (func(data datasource.Data) error, error) {
+			oldName := in.Name()
+
+			if err := in.Rename(oldName + "_raw"); err != nil {
+				return nil, fmt.Errorf("renaming field: %w", err)
+			}
```

Comment:
I know we've used this behavior for backwards compatibility, but also had some discussions around how to improve naming. The issue is that the field type would change depending on whether this operator is present or not.

/cc @mauriciovasquezbernal @alban